src/Hyperlink/index.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import classNames from 'classnames';
+import FontAwesomeStyles from 'font-awesome/css/font-awesome.min.css';
+import PropTypes from 'prop-types';
+import isRequiredIf from 'react-proptype-conditional-require';
+
+function Hyperlink(props) {
+  const {
+    destination,
+    content,
+    target,
+    onClick,
+    externalLinkAlternativeText,
+    externalLinkTitle,
+    ...other
+  } = props;
+
+  let externalLinkIcon;
+
+  if (target === '_blank') {
+    externalLinkIcon = (
+      // Space between content and icon
+      <span>{' '}
+        <span
+          className={classNames(FontAwesomeStyles.fa, FontAwesomeStyles['fa-external-link'])}
+          aria-hidden={false}
+          aria-label={externalLinkAlternativeText}
```

Comment:
This `aria-label` field is meaningful because for screen-readers it will display the text (e.g. "open in a new tab") instead of a less useful `[?]` value that's supposed to represent the icon.